taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from .models import Driver, Manufacturer, Car
 
-# Create your views here.
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_cars = Car.objects.count()
+    num_Manufacturers = Manufacturer.objects.count()
```

Comment:
The variable name 'num_Manufacturers' does not follow Python's standard naming conventions. Variable names should be in lowercase, with words separated by underscores as necessary to improve readability. Consider renaming it to 'num_manufacturers'.